### DISCUSSION TITLE
`metadata.type` is deprecated in favor of the global `metricType` for CPU, Memory, Datadog scalers

### DISCUSSION THREAD

**Discussion by tomkerkhove** [kedacore/keda#3002] [Deprecations]
As of KEDA v2.7, `metadata.type` is deprecated in favor of the global `metricType` and will be removed in KEDA v3.0.

Learn more on https://github.com/kedacore/keda/issues/2030

**Comment by zmei-ta** [comment]
running into the following error in `keda.sh/v1alpha1` when I run the helm upgrade command

```Error: UPGRADE FAILED: error validating "": error validating data: ValidationError(ScaledObject.spec.triggers[0]): unknown field "metricType" in sh.keda.v1alpha1.ScaledObject.spec.triggers```

**Reply by tomkerkhove** [reply]
To what version are you migrating? @JorTurFer Did we remove this field by accident?

**Reply by JorTurFer** [reply]
I don't think so, the conflict is with the new property. I think KEDA isn't updated or CRDs aren't properly updated.
What KEDA version are you running @zmei95 ?

**Reply by zmei-ta** [reply]
hope this help
```
NAME                                    	CHART VERSION	APP VERSION	DESCRIPTION
kedacore/external-scaler-azure-cosmos-db	0.1.0        	0.1.0      	Event-based autoscaler for Azure Cosmos DB chan...
kedacore/keda                           	2.8.2        	2.8.1      	Event-based autoscaler for workloads on Kubernetes
kedacore/keda-add-ons-http              	0.3.0        	0.3.0      	Event-based autoscaler for HTTP workloads on Ku...

**Reply by JorTurFer** [reply]
no, I need the version you are running in the cluster where you have the problems :) 
This command should show the image tag with the version 
`kubectl get deploy keda-operator -n keda -o=jsonpath='{$.spec.template.spec.containers[:1].image}'`
Could you share the output from that command?
